devops/sandbox/values.yaml

```diff
@@ -60,9 +60,12 @@ iam-cache-server-helm:
     encryptedData:
       SENTRY_DNS: AgB6Cjvjl4ykIZAadTyJSKH/QbkVN9QFTDrIPyevC2aOjU30EQDospRyd19KT6VuSA9O748/iCxZzpuwtzp2toHvzTaio211bFKDdkXi/FyHfOSLAGtx3c+vfot0ZC+TsCJU+5C7mGLAfeRo5GaRTxA4yCPSIt3Bz9/QmRnyTXW+LP/pg19ERIJCLs8D3osd3av79xfmNID4EkTnFqKCKKTju2cEXd4Vh87dXX6oq0Cd2LJ8F25J/gE2NKJsGkQAiNOToZChbjXsTU1WL/JwNaH+CydhdDt7A+GGlGcByvEpBTA5d+MkDm+bTCMe5pMuysuUbI49VyxYkNMIsXRbfPd8CezzGPd54BcI79opPEWe+xcBo3odPIoGP+2BwyEcoyPSa0SZ1wMHmzgyevDIOUS3ktRn6luTvtTDGbZoV9g9qKMRrgk2fYfH81yPNClR+AAUa/L09A83rqUSsKFP7fQIBb5nUjy8SPRh6hFyA5dFk+LMJGwB4q/NhPPEl2zmoP84VNqOGsXSTZjI7lrhy6HTYfN2gj2YNLHILHCggZn+nMOEeZLRczRX8OlnVzcSITZICe7/cWyynIsEY7p3HdYJ+k0iVZUe3ySxGQEFvWhUkXWUBYC1kN34G3R+TUxFdOgBoGcJKGoDXAHw5cDrW2LcQ2Um6T7b49Ta+NH7yJdLzRYQ1yuVs2E16oiM2GXHfQXpA0RJ8qlSFY8j10+8FNMPUsnSslHaSzF8dKzs+y8yBD+xE+vi3CGSeesBUz/xPnGYY6WiCsAREIu0KZp8vgLT5yDSlfhM94i0jQ==
       STRATEGY_PRIVATE_KEY: AgBM4BIbZIvZ9ErUDLWmmgw0UMEGk4k9hO54/tos4Rf9bUeewlDyAOrnz+gcl8uXuwFiGazQYNia2JFOjCapKnVt0h3LUjvubN4G2cVoQQl9E2+BDzSb7Ucji2UsdleN1ScOwd6XOADdw7Pa1LUXdpf32rf5YADhdYLGDiYy6hqpJ0OPd+61s92HgtyKHioTSP3fJ4YyFqgXQ4wORRR15rdwJdvzSxUrzGnft/ceuB+ilGJc0qSu/jT+SifQ1411zZ6/vsvhfvSq7q4wXEibn2OWSy7MxxNdYafVC7PbyFgc9i6t8V2YKJzUigIGV53ds9GuMK8i8CeGVg2R3e4iHyuOuWv6LeNjbdUd6QeAynOFVos9rBWwS/L7g+Rv0aUUnD016GYSoPwLGPRwvpyhk3jRtJ94uj4FAkUwWGWKagTA+F0uZyIjq2emO4opE8J5FwbPJOA3SKxfo0yPxgQF7S7uTAGlslQdxSrBk9ICUb5FuMVZ32Bxv3fbJ4R3xIS6D8g9hfPG3/P+zpvTxMzPlOjn7bgXHpbV6qOtt47OjTb9A+ELEVnwF/hOY6efSNa/MYlDU0wjF8CGveKFmYkSo9QHnTx7ruGu1s18yXMQZGK47WEERFR3WS0JqB7iQKv2KhPlMJLQapbROLP3MEYGWpEExDdfQRksmvCpQvQhM/7rtBpjobktxKfP6FVsiCTiL0A8Lkbo+i48oVti3SK6vNF2kDBhy4b6uLWwztYNvz3XL1ZalmcWSwe6iMtscD8Vcjms4FVe3hBRPZbXK5/wOpMq
+      ENS_URL: AgDogSHI4CvlKQEZAeop8qWQ4isVl9prSFdz7VDvSCJZ2M9cBsKfYcKFTZwuxznxGJ6vfNT9gpmNgB2R9sHvv+f0sx7ngK5noqQheAGvA6R5jks/5N+digvPrMAc8bZdrlo1WTedhKThJ/eM5S5jNQnBKbFeoUIrlUWvSRZWDYjMnzJm2HU1SCnO9hni6o0OUVde242b0c+AzKMCMHXTQ9wR5F3u0URuk8RguTScBaREN5brQeZs5hTTrAUZlqAc92an9khQhB8t0JEn4fc4Z4p97LuCVsVsA60GlcAGPy2hLOJXL2Qg7Q3k34ralmDCQBnby7dLFOL2aja0DtcKvhQYmoPlQyS4C1AHWZi8jOWCf9dzEYvDtsf/pkb5wdoqq7DblCsyRSN/w656kIsd992mbhIJMNeIKW64WdYQJ586VQZVxSBhKogXQISfDigHv7BhlyilBbQqHNEppdLNNTPvKhIX/M/efTgdlP0WcXCYWJL8UMW9afQTcS9lSf9od6/HcF2RRt4Lg/PSith2vsj/XOis+UAVIQRcC6ozqV8UVc4vqtrfkdL47MWppCljybbp0kQvKE/dLcXOre+/6ezQHcp8XDJYXWlcwbwuVMDr/W4qohydE1OAOdSznD+EiEq1Uvc178T44qK2QoQ0xan9okleU9UKl/yUKRxI1qnQswcMNNKciOdD43ikrMxfT/nFhi3MiMjec0av+Tx7EK4OuwXf5o8UGEwtspoIo1ftBxf8M858u2DkgZSMQ/KwXauuiExRkA==
```

Comment:
@JGiter were you able to add this yourself in the end or a devops team member helped out?

Reply:
I copied this from sandbox branch https://github.com/energywebfoundation/ssi-hub/blob/85120d71f5be204b9feed81879d81ad60ca1185b/devops/sandbox/values.yaml#L63. It should be correct I suppose?